velox/exec/tests/AggregationFuzzerTest.cpp

```diff
@@ -20,15 +20,17 @@
 #include <unordered_set>
 
 #include <boost/random/uniform_int_distribution.hpp>
-#include "velox/exec/tests/utils/AggregationFuzzerRunner.h"
+#include "velox/exec/fuzzer/AggregationFuzzerOptions.h"
+#include "velox/exec/fuzzer/AggregationFuzzerRunner.h"
+#include "velox/exec/fuzzer/DuckQueryRunner.h"
 #include "velox/exec/tests/utils/AssertQueryBuilder.h"
-#include "velox/exec/tests/utils/DuckQueryRunner.h"
 #include "velox/exec/tests/utils/PlanBuilder.h"
 #include "velox/functions/prestosql/aggregates/RegisterAggregateFunctions.h"
+#include "velox/functions/prestosql/fuzzer/ApproxDistinctInputGenerator.h"
```

Comment:
Thank you for organizing code nicely. Would you update PR description to explain how the new code layout?